librz/util/buf.c

```diff
@@ -310,7 +341,20 @@ RZ_API RzBuffer *rz_buf_new_mmap(const char *filename, int perm, int mode) {
 	return new_buffer(RZ_BUFFER_MMAP, &u);
 }
 
-RZ_API RzBuffer *rz_buf_new_slice(RzBuffer *b, ut64 offset, ut64 size) {
+/**
+ * \brief Creates a new buffer from a slice of another buffer.
+ * \param b The source buffer used to create the sub buffer.
+ * \param offset The starting offset in the source buffer.
+ * \param size The number of bytes that will be extract from the source buffer.
+ * \return Return the new allocated buffer.
+ *
+ * \see rz_buf_ref()
+ *
+ * The function creates a new allocated buffer using the RZ_BUFFER_REF as back end.
+ * The allocated buffer will reference the source buffer. So both buffer are
+ * synchronize
```

Comment:
The function creates a new buffer which shows just a slice of another one, passed as argument. The allocated buffer will reference the source buffer. So both buffer are synchronized.